scripts/edit_media_directories.py

```diff
@@ -20,25 +31,43 @@ def get_usb_drives(logger: logging.Logger):
 
 def edit_directories(logger: logging.Logger):
   """Update LMS server with list of drives"""
-  usb_drives = get_usb_drives(logger) # Get list of available drives
-  usb_drives.append("") # Add blank drive to reflect empty string that is always at the end of the mediadirs section of request body
-  url = 'http://localhost:9000/settings/server/basic.html'
-  data={
-    'saveSettings': '1',
-    'useAJAX': '0',
-    'page': 'BASIC_SERVER_SETTINGS',
-    'pref_language': 'EN',
-    'pref_libraryname': '',
-    'pref_playlistdir': '',
-    'pref_rescantype': '1rescan'
-  } # All default headers, mediadirs added in loop
-  for d, drive in enumerate(usb_drives):
-    data[f"pref_mediadirs{d}"] = drive
-    data[f"pref_ignoreInAudioScan{d}"] = 1
-
-  logger.info("Adding drives to LMS settings...")
-  requests.post(url, data=data, verify=False, timeout=5)
-  logger.info("Drives added, LMS scanning now.")
+  try:
+    usb_drives = get_usb_drives(logger) # Get list of available drives
+    LMSMode = CheckLMSMode()
```

Comment:
This assignment reads confusing it looks like you are assigning a class/type from an instantiated class.